brut.apktool/apktool-lib/src/main/java/brut/androlib/Androlib.java

```diff
@@ -486,6 +488,25 @@ public boolean buildResourcesFull(File appDir, UsesFramework usesFramework)
                     }
                 }
 
+                if (buildOptions.netSecConf) {
+                    MetaInfo meta = readMetaFile(new ExtFile(appDir));
+                    if (meta.sdkInfo != null && meta.sdkInfo.get("targetSdkVersion") != null) {
+                        if (Integer.parseInt(meta.sdkInfo.get("targetSdkVersion")) < 24) {
```

Comment:
Can we use the enum in `ResConfigFlags.SDK_NOUGAT` for this 24?

Reply:
done